DISTRIBUTE

```diff
@@ -56,21 +57,29 @@ Clear out old builds:
 Be sure that LD_LIBRARY_PATH and similar aren't set so that the conda
 environment isn't pulling in anything from the host environment.
 
-Build so getting the latest inputs for the autodocs:
-    PYTHONNOUSERSITE=1 PYTHONPATH= python setup.py build
+    PYTHONNOUSERSITE=1 PYTHONPATH= python-build -s -n -x
```

Comment:
Should the date at the top of `DISTRIBUTE` also be updated?

Reply:
Yep, definitely should. Fixed and pushed.